### ISSUE TITLE
bug: Rotating from portrait to landscape breaks scanner on ipad

### ISSUE THREAD

**Issue by mbrunneritlabs** [capawesome-team/capacitor-mlkit#168]
### Plugin(s)

- [X] Barcode Scanning
- [ ] Face Detection
- [ ] Face Mesh Detection
- [ ] Selfie Segmentation
- [ ] Translation

### Version

6.1.0

### Platform(s)

- [ ] Android
- [X] iOS
- [ ] Web

### Current behavior

On most devices when opening the barcode scanner in portrait mode and then turning to landscape mode the picture in the scanner window turns correctly.

On Ipad10 when turning the barcode scanner window from portrait to landscape the picture doesnt turn and the window is cut on the right side. 

So our app itself and the overlay turns correctly to landscape mode but the scanner stays in portrait mode. 

// Portrait Mode

![Image](https://github.com/user-attachments/assets/2467f255-259f-4b4d-9942-2dd34306e005)

// Broken Landscape Mode after rotating

![Image](https://github.com/user-attachments/assets/fd350807-ed68-42a9-b307-fee99d0cfa46)

Interesting:

Starting the scanner in landscape mode works correctly. The issue only occurs when turning the scanner from portrait to landscape while running and only on our ipad test device. 

### Expected behavior

When turning to landscape mode the scanner turns correctly on ipad.

### Reproduction

The bug lies withing the scanner itself

### Steps to reproduce

Run BarcodeScanner.startScan on Ipad 10
Turn from Portrait Mode to Landscape -> The picture from camera is rotated incorrectly and black bar occurs. 

### Capacitor doctor

I currently cannot run this command but can provide the next days. 

### Before submitting

- [X] I have read and followed the [bug report guidelines](https://capawesome.io/contributing/bug-reports/).
- [X] I have attached links to possibly related issues and discussions.
- [X] I understand that incomplete issues (e.g. without reproduction) are closed.

**Comment by robingenz** [comment]
Unfortunately, I don't have access to an iPad 10. Feel free to create a PR.

**Comment by orl99** [comment]
are we progressing in this?

**Comment by anusha-kaparapu** [comment]
Hi @robingenz we are also facing the same issue in ipad devices, we tried workaround but that is causing some camera glitchi, it is not working as expected. please find the below workaround code.

```
setTimeout(() => {
	ScreenOrientation.addListener('screenOrientationChange', (res: ScreenOrientation) => {
		this.ngZone.run(() => {
			this.screenOrientationTypeChange = res?.type;
			if (this.screenOrientationTypeChange === 'landscape-primary' ||
				this.screenOrientationTypeChange === 'landscape-secondary' ){
					console.log('start scanner');
					this.startScanner().catch(() => {});
			}
		});
	});
}, 100);

```
Is there any plan to fix this issue for upcoming releases?

**Comment by robingenz** [comment]
This is fixed in the latest version, see https://github.com/capawesome-team/capacitor-mlkit/commit/dd12ea8b8f5906f0ab56f3c1672d09e5a48426bb.

**Comment by robingenz** [comment]
@anusha-kaparapu Yes, you're right. Something went wrong with the merge. I have created a new PR: #248. Thanks for letting me know!

**Comment by anusha-kaparapu** [comment]
Thanks for quick response @robingenz , when can we expect these changes in latest 7.0.0 version as we are moving to production soon. i have one more dought according to the document, the capacitor core should be 7, but we are only utilising 6 because of project requirements.
We used the following command to force the package upgrade.
1.  npm install @capacitor-mlkit/barcode-scanning@7.0.0 —force
2. Will there be any problems pushing to app stores or will there be any issues with the packages I forced to update?

**Comment by robingenz** [comment]
@anusha-kaparapu 

> when can we expect these changes in latest 7.0.0 version as we are moving to production soon.

We plan to publish a new release this week.

> Will there be any problems pushing to app stores or will there be any issues with the packages I forced to update?

I can't answer that as I have never tested this combination.

**Comment by anusha-kaparapu** [comment]
Thanks @robingenz

**Comment by anusha-kaparapu** [comment]
Hi @robingenz we have upgraded to latest 7 version of barcode scanner and try to build an archive my app in xcode and try to export privacy report.

![Image](https://github.com/user-attachments/assets/0b414542-88a3-4089-a9e4-43922082a484)

 but still im getting below report

![Image](https://github.com/user-attachments/assets/2c1623b4-98e2-4ca5-a49b-49197317ea3a)

but i got below email from Apple store 

![Image](https://github.com/user-attachments/assets/9dfeb1d2-9990-414f-b2be-1aacdac591ac)

inorder to cross verify I uninstalled barcode-scanner then  my privacy report is empty(no issues). im confuse here, how to solve both issues please guide me on it

**Comment by anusha-kaparapu** [comment]
@robingenz when can we expect these commit changes https://github.com/capawesome-team/capacitor-mlkit/pull/248, as we are close to our production release.

**Comment by robingenz** [comment]
@anusha-kaparapu We will publish the next release today.

**Comment by anusha-kaparapu** [comment]
Thanks @robingenz
